include/TrustWalletCore/TWBlockchain.h

```diff
@@ -56,6 +56,7 @@ enum TWBlockchain {
     TWBlockchainEverscale = 42,
     TWBlockchainAptos = 43, // Aptos
     TWBlockchainHedera = 44, // Hedera
+    TWBlockchainTON = 46,
```

Comment:
```suggestion
    TWBlockchainTON = 45,
```

Reply:
you may need to clean the build folder to avoid some random iOS / Android failures